legacy/jacorb/src/main/java/org/jboss/as/jacorb/JacORBExtension.java

```diff
@@ -68,6 +70,8 @@
 
     private static final ModelVersion CURRENT_MODEL_VERSION = ModelVersion.create(1, 4, 0);
```

Comment:
EAP 6.2 used ModelVersion 1.3.0, so transforming to 1.2.x and earlier can be dropped.

Note that parsing previous xsds can't be dropped; that's a different thing.